package.json

```diff
@@ -41,9 +41,13 @@
   },
   "homepage": "https://github.com/RedHatInsights/insights-chrome#readme",
   "devDependencies": {
+    "@babel/core": "^7.3.3",
```

Comment:
If we are using new `babel` can we also update all babel deps to version 7? Or create an issue to mark it and we should update in follow-up PR

Reply:
Let's create an issue to work on it in a follow-up